translations/monero-core_fr.ts

```diff
@@ -568,217 +543,212 @@
         <translation>Entrez le mot de passe du portefeuille</translation>
     </message>
     <message>
-        <location filename="../components/PasswordDialog.qml" line="88"/>
-        <source>Please enter wallet password for:&lt;br&gt;</source>
-        <translation>Entrez le mot de passe du portefeuille pour :&lt;br&gt;</translation>
+        <location filename="../components/PasswordDialog.qml" line="86"/>
+        <source>Please enter wallet password for: </source>
+        <translation>Veuillez entrer votre mot de passe pour: </translation>
     </message>
     <message>
-        <location filename="../components/PasswordDialog.qml" line="156"/>
+        <location filename="../components/PasswordDialog.qml" line="150"/>
         <source>Cancel</source>
         <translation>Annuler</translation>
     </message>
     <message>
-        <location filename="../components/PasswordDialog.qml" line="169"/>
+        <location filename="../components/PasswordDialog.qml" line="161"/>
         <source>Continue</source>
         <translation>Continuer</translation>
     </message>
 </context>
 <context>
     <name>PrivacyLevelSmall</name>
     <message>
-        <location filename="../components/PrivacyLevelSmall.qml" line="100"/>
+        <location filename="../components/PrivacyLevelSmall.qml" line="96"/>
         <source>Low</source>
         <translation>Bas</translation>
     </message>
     <message>
-        <location filename="../components/PrivacyLevelSmall.qml" line="109"/>
+        <location filename="../components/PrivacyLevelSmall.qml" line="105"/>
         <source>Medium</source>
         <translation>Moyen</translation>
     </message>
     <message>
-        <location filename="../components/PrivacyLevelSmall.qml" line="118"/>
+        <location filename="../components/PrivacyLevelSmall.qml" line="114"/>
         <source>High</source>
         <translation>Haut</translation>
     </message>
 </context>
 <context>
     <name>ProgressBar</name>
     <message>
-        <location filename="../components/ProgressBar.qml" line="41"/>
-        <source>Establishing connection...</source>
-        <translation>Établissement de la connexion...</translation>
-    </message>
-    <message>
-        <location filename="../components/ProgressBar.qml" line="58"/>
-        <source>Blocks remaining: %1</source>
-        <translation>Blocs restants : %1</translation>
+        <location filename="../components/ProgressBar.qml" line="38"/>
+        <source>%1 blocks remaining: </source>
+        <translation>%1 blocs restants</translation>
     </message>
     <message>
-        <location filename="../components/ProgressBar.qml" line="109"/>
-        <source>Synchronizing blocks</source>
-        <translation>Synchronisation des blocs</translation>
+        <location filename="../components/ProgressBar.qml" line="70"/>
+        <source>Synchronizing %1</source>
+        <translation>Synchronisé %1</translation>
```

Comment:
Posting this here because I can read French, but it's probably a confusing issue for all languages: the %1 placeholder in `%1 blocks remaining: ` and `Synchronizing %1` does not represent a number of blocks, but rather the string `syncType`, which is either Wallet or Daemon. See the GUI code in components/ProgressBar.qml. The number of blocks remaining will be added after `%1 blocks remaining: `. Relevant code fragments:

``` 
    property string syncType // Wallet or Daemon
    property string syncText: qsTr("%1 blocks remaining: ").arg(syncType)
```

```
                progressText.text = syncText + remaining.toFixed(0);
```

```
            text: qsTr("Synchronizing %1").arg(syncType)
```